pandas/tests/dtypes/test_dtypes.py

```diff
@@ -1212,3 +1212,8 @@ def test_multi_column_dtype_assignment():
 
     df["b"] = 0
     tm.assert_frame_equal(df, expected)
+
+
+def test_empty_object_array_is_string_dtype():
```

Comment:
hmm looks like we have tests for is_string_dtype in a couple of places.  one of these days might be worth checking if there is a reason for that and if not, refactoring

Reply:
Thank you for the comment. I think, I found a better place for my test: `pandas/tests/dtypes/test_common.py`. I combined my test with another one and added a parameterization.